Sources/JavaScriptKit/BasicObjects/JSError.swift

```diff
@@ -13,6 +13,10 @@ public final class JSError {
     public var name: String {
         ref.name.string!
     }
+
+    public var stack: String {
```

Comment:
I think it would be more safe if non-standard field is exposed as `String?`. For example, a new JS engine may not implement this feature.